com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ModelUtil.java

```diff
@@ -670,48 +670,56 @@ private static void findModelNodes(Node node, List<Node> modelNodes) {
         }
     }
 
+    private static void calcCenterNode(Node node, Aabb aabb) {
+        if (node.model != null) {
+            // As a default, we only count nodes with models, as the user
+            // cannot currently see/use the lights or cameras etc that are present in the scene.
+            aabb.expand(node.world.translation.x, node.world.translation.y, node.world.translation.z);
+        }
+
+        for (Node child : node.children) {
+            calcCenterNode(child, aabb);
+        }
+    }
+
+    // Currently finds the center point using the world positions of each node
     private static ModelImporter.Vec4 calcCenter(Scene scene) {
-        ModelImporter.Vec4 center = new ModelImporter.Vec4(0.0f, 0.0f, 0.0f, 0.0f);
-        float count = 0.0f;
+        Aabb aabb = new Aabb();
         for (Node root : scene.rootNodes) {
-            ArrayList<Node> modelNodes = new ArrayList<>();
-            findModelNodes(root, modelNodes);
-
-            for (Node modelNode : modelNodes) {
-                center.x += modelNode.local.translation.x;
-                center.y += modelNode.local.translation.y;
-                center.z += modelNode.local.translation.z;
-                count++;
```

Comment:
This is not how to calculate a center point, using a flattened tree and local points?! (not sure what I was thinking)